Sources/TestSupport/misc.swift

```diff
@@ -86,21 +80,25 @@ public func fixture(name: String, tags: [String] = [], file: StaticString = #fil
     }
 }
 
-/// Test-helper function that creates a new Git repository in a directory.  The new repository will contain exactly one empty file, and if a tag name is provided, a tag with that name will be created.
-public func initGitRepo(_ dir: AbsolutePath, tag: String? = nil, file: StaticString = #file, line: UInt = #line) {
-    initGitRepo(dir, tags: tag.flatMap{ [$0] } ?? [], file: file, line: line)
+/// Test-helper function that creates a new Git repository in a directory.  The new repository will contain
+/// exactly one empty file unless `addFile` is `false`, and if a tag name is provided, a tag with that name will be created.
+public func initGitRepo(_ dir: AbsolutePath, tag: String? = nil, addFile: Bool = true, file: StaticString = #file, line: UInt = #line) {
+    initGitRepo(dir, tags: tag.flatMap{ [$0] } ?? [], addFile: addFile, file: file, line: line)
 }
 
-public func initGitRepo(_ dir: AbsolutePath, tags: [String], file: StaticString = #file, line: UInt = #line) {
+public func initGitRepo(_ dir: AbsolutePath, tags: [String], addFile: Bool = true, file: StaticString = #file, line: UInt = #line) {
     do {
-        let file = dir.appending(component: "file.swift")
-        try systemQuietly(["touch", file.asString])
+        if addFile {
+            let file = dir.appending(component: "file.swift")
+            try systemQuietly(["touch", file.asString])
+        }
+
         try systemQuietly([Git.tool, "-C", dir.asString, "init"])
         try systemQuietly([Git.tool, "-C", dir.asString, "config", "user.email", "example@example.com"])
         try systemQuietly([Git.tool, "-C", dir.asString, "config", "user.name", "Example Example"])
         try systemQuietly([Git.tool, "-C", dir.asString, "config", "commit.gpgsign", "false"])
-        try systemQuietly([Git.tool, "-C", dir.asString, "add", "."])
-        try systemQuietly([Git.tool, "-C", dir.asString, "commit", "-m", "msg"])
+        try addGitRepo(dir, file: RelativePath("."))
+        try commitGitRepo(dir, message: "msg")
```

Comment:
We should get rid of these global methods in favour of this https://github.com/apple/swift-package-manager/blob/master/Sources/TestSupport/GitRepositoryExtensions.swift